### ISSUE TITLE
[model] Agda 2.6.0

### ISSUE THREAD

**Issue by jonsterling** [freebroccolo/yggdrasil#22]
I'm having trouble building the model using Agda 2.6.0. I get the following error:

```
/Users/jon/src/yggdrasil/docs/model/src/Main.agda:197,28-32
ϰ₀ != ϰ₁ of type (List Face)
when checking that the pattern refl has type
((ϡ₀ ⊸ ϰ₀) ≡ (ϡ₁ ⊸ ϰ₁))
```

**Comment by jonsterling** [comment]
(Maybe I should just downgrade... I've noticed some problems with pattern matching in this version of Agda.)

**Comment by jonsterling** [comment]
I see how to fix these and make them work in Agda 2.6.0; I might just go through and do so, as it's not to terrible.
